networkconfig/holesky-e2e.go

```diff
@@ -10,7 +10,8 @@ import (
 var HoleskyE2E = NetworkConfig{
 	Name:                 "holesky-e2e",
 	Beacon:               beacon.NewNetwork(spectypes.HoleskyNetwork),
-	Domain:               spectypes.DomainType{0x0, 0x0, 0xee, 0x1},
+	GenesisDomainType:    spectypes.DomainType{0x0, 0x0, 0xee, 0x0},
+	AlanDomainType:       spectypes.DomainType{0x0, 0x0, 0xee, 0x1},
```

Comment:
probably `CurrentDomainType` and `NextDomainType` are better names and can be used in future forks as well

Reply:
You're right in essence, although I think it could be useful to give the domain type the name of the fork, so we can later add a 3rd one (or just rename the Alan one)